bats/fb-test-katello.bats

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
```

Comment:
ack. Was not sure about the history and thought porting was cheaper then thinking.

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_